### ISSUE TITLE
dungeon finder fix for wotlk 3.3.5

### ISSUE THREAD

**Issue by Naylz** [celguar/mangosbot-bots#359]
I  know that it has been discussed in previous posts that the dungeon finder does not work in wotlk. however, I have noticed that even when I create a group by myself, the bots do not accept their roles correctly. unfortunately.  this makes the dungeon finder totally superfluous. is there a fix planned or does anyone have a workaround? it is tedious to always run to every dungeon and kills a bit of the otherwise really wonderful wotlk experience. 

still the best server ever. thanks for the hard work!

**Comment by celguar** [comment]
Hello, bots accept their roles based on talents/strategies. If you want a tank, you need to get a protection specced warrior. not just some random one

**Comment by Naylz** [comment]
yes I know and thats exactly how I do it. Im a tank, invite 2 dd bots and one healer bot. I configure the bots with the mangosbot addon that each role is assigned correctly. then I que the group in dungeon browser and it fails at role check. :/
any ideas?
maybe the bug causes by another addon?

**Comment by celguar** [comment]
they use talents as well. ALso if you invite a prot warrior with a shield and use addon to tell him to do DPS he will suck at it, because he has a shield and tank talents. same for healer. You need to pick bot that are ALREADY tank or dps or heal. Whisper bot "who" to know their spec. Or use chat channels to find some bots. e.g. /1 @dps who

**Comment by Naylz** [comment]
thanks so much. didn´t know this. you are genius. ;)

**Comment by rp-ukr** [comment]
I go into dungeons closes mangosd.exe (wotlk 3.3.5)
